src/teaching-element/Accordion/Item.vue

```diff
@@ -4,8 +4,11 @@
       <div class="contents">
         <span class="title">{{ heading }}</span>
       </div>
-      <slot name="accordionHeaderIcon">
-        <span class="icon"></span>
+      <slot v-if="!expanded" name="accordionHeaderClosedIcon">
+        <span class="closed-icon"></span>
+      </slot>
+      <slot v-if="expanded" name="accordionHeaderOpenIcon">
+        <span class="open-icon"></span>
```

Comment:
A couple of things:

1. Is there a reason why you did not write a` v-if`/`v-else` combo here? Is there an issue with it on slots?
2. The expansion is something that can happen frequently or not at all. It truly depends on the user. Vue [documentation](https://vuejs.org/v2/guide/conditional.html#v-if-vs-v-show) recommends the usage of `v-show` in cases where toggling is used often. Since I can't really anticipate how often this will happen I recommend one of these two solutions:

    1. Keep `v-if` but wrap them inside a `keep-alive` component
    2. Replace with `v-show`

I believe both of these solutions are good enough, maybe `v-show` is more standard and in line with the way it is written now.

Reply:
> Is there a reason why you did not write a v-if/v-else combo here?

General sloppiness. I'll replace it with `v-show`.